src/vmm/src/builder.rs

```diff
@@ -84,7 +84,7 @@ pub enum StartMicrovmError {
     CreateLegacyDevice(device_manager::legacy::LegacyDeviceError),
     /// Memory regions are overlapping or mmap fails.
     #[error("Invalid Memory Configuration: {}", format!("{:?}", .0).replace('\"', ""))]
-    GuestMemoryMmap(crate::vstate::memory::Error),
+    GuestMemoryMmap(crate::vstate::memory::VmMemoryError),
```

Comment:
Although this is a common practice for us to move away from `Error`, it is worth mentioning in the commit message.

Reply:
fixed